Cargo.toml

```diff
@@ -67,6 +67,7 @@ rocksdb = "0.21.0"
 rustls = { version = "0.23.23", default-features = false }
 reqwest = { version = "0.12.9", features = ["multipart"] }
 rstest = "0.25.0"
+schemars = "0.8.22" # This version needs to be exactly the same as in `near_sdk::schemars`
```

Comment:
After reading around a lot, I think we don't need to use `schemars` directly. We should use instead only `near-sdk/schemars` and only when the target is not wasm32. Please correct me if I am wrong @kuksag